### DISCUSSION TITLE
minimap

### DISCUSSION THREAD

**Discussion by TLovers** [antvis/X6#2200] [Q&A]
看文档上说小地图是完整画布的预览，支持通过平移缩放小地图的视口来平移缩放画布。在 v1.23.x 版本之前，小地图需要配合 scroller 使用， 但是依赖装的是1.32.3为什么不配置scroller 还是不能正常展示？
https://codesandbox.io/s/hungry-franklin-hl965s?file=/src/components/test.vue

**Comment by NewByVector** [comment]
具体的问题是什么呢？
